cmd/telemetrygen/internal/logs/exporter.go

```diff
@@ -28,24 +29,32 @@ func newExporter(ctx context.Context, cfg *Config) (exporter, error) {
 		}, nil
 	}
 
-	if !cfg.Insecure {
-		return nil, fmt.Errorf("'telemetrygen logs' only supports insecure gRPC")
+	// TODO: Use an otlplogs grpc package in the future
```

Comment:
Can you file an issue for this, explaining why we can't do this yet?

Reply:
I wasn't able to find such a package. Does it exist?

Reply:
It does not, I just want to have an issue in this repository tracking this work, since TODOs are often missed :) You can just say "We should switch to opentelemetry-go logs exporter once available, see https://github.com/open-telemetry/opentelemetry-go/issues/4696 for the current state"